packages/integrations/gei-notifications/schema.graphql

```diff
@@ -0,0 +1,271 @@
+type Query{
+	userQuery(
+		userId: String!
+	): UserQuery
+	publicQuery: PublicQuery!
+}
+
+type UserQuery{
+	listNotifications(
+		input: ListNotificationsInput
+	): ListNotificationsResult!
+	listChannels(
+		input: ListChannelsInput
+	): ListChannelsResult!
+	generatePushNotificationToken: GeneratePushNotificationTokenResult!
```

Comment:
```suggestion
	generatePushNotificationToken: GeneratePushNotificationTokenResult!
```

Dlaczego to jest w query? Może aktualna implementacja nie ma żadynch side effectów, ale sama nazwa `generate` sugeruje, że kiedyś będą potrzebne.